app/assets/stylesheets/hyrax/_styles.scss

```
@@ -67,3 +67,12 @@ label.disabled {
   font-size: 1.2em;
   margin: 0 0 10px;
 }
+
+
+// Bootstrap 4 has no default row margins, so this helper
+// class will wrap rows giving some margin between
+.spaced-rows {
+  .row+.row {
+    margin-top: 1rem;
+  }
```

This works well until we get to smaller viewports and the wrapping row's vertical spacing doesn't matter as much as a columns. What if it was done this way? 

```suggestion
  .row > .col {
    margin-bottom: 1rem;
  }
```